### ISSUE TITLE
Breakpoin doesn't work inside for loop

### ISSUE THREAD

**Issue by ktisha** [sproshev/TheRPlugin#37]
packageNames <- .packages(all = TRUE)

for (name in packageNames) {
    shouldLoadLibrary = FALSE
}

**Comment by sproshev** [comment]
Currently `for loop` is not supported in `main script` due to code submission.
Debugger send `main script` line by line to interpreter and when `for loop` has been completely submitted interpreter runs it without any debug information.

Possible workarounds:

```
foo <- function() {
    packageNames <- .packages(all = TRUE)

    for (name in packageNames) {
        shouldLoadLibrary = FALSE
    }
}

foo()
```

and 

```
foo <- function(name) {
    shouldLoadLibrary = FALSE
}

packageNames <- .packages(all = TRUE)

for (name in packageNames) {
    foo(name)
}
```

**Comment by sproshev** [comment]
Fixed in af94477fc9645db592754fd0044a466a5ab2995d.
